bundler/spec/commands/binstubs_spec.rb

```diff
@@ -241,7 +241,7 @@
         it "falls back to the latest installed bundler" do
           FileUtils.rm bundled_app_lock
           sys_exec "bin/bundle install", :env => { "DEBUG" => "1" }
-          expect(out).to include "Using bundler #{system_bundler_version}\n"
+          expect(out).to include "Using bundler "
```

Comment:
This is not the spec that's failing, it's the one at line 229, right?

Reply:
To fix the spec in line 229, the fix is not good because if we change the expectation, then the spec no longer checks what it's supposed to check (that bundler actually "trampolines" to a higher version). Instead we should parse the version in the "Using bundler x.y.z\n" message, and check that the version is `>= system_bundler_version`. I think that would get the spec to pass on ruby's stable branch while still testing what we want to test here.

Reply:
Oh,,, sorry yes you are right..

Reply:
> Oh,,, sorry yes you are right..

Rebased with the line 229 modification.

> To fix the spec in line 229, the fix is not good because if we change the expectation, then the spec no longer checks what it's supposed to check (that bundler actually "trampolines" to a higher version). Instead we should parse the version in the "Using bundler x.y.z\n" message, and check that the version is >= system_bundler_version. I think that would get the spec to pass on ruby's stable branch while still testing what we want to test here.

That's a good idea. Let me try to update this PR.